package.json

```diff
@@ -13,6 +13,7 @@
     "clean": "gatsby clean",
     "version": "tsc -p .",
     "migrate": "DOTENV_CONFIG_PATH=.env.development ts-node -r dotenv/config.js ./node_modules/node-pg-migrate/bin/node-pg-migrate -j ts -m src/migrations -d CONNECTION_STRING",
+    "create": "DOTENV_CONFIG_PATH=.env.development ts-node -r dotenv/config.js ./node_modules/node-pg-migrate/bin/node-pg-migrate -j ts -m src/migrations create",
```

Comment:
Please remove this, should use `npm run migrate create` instead

Reply:
ty, that's what I needed